src/app/components/Figure/Copyright/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import {
+  FF_NEWS_SANS_REG,
+  C_WHITE,
+  C_EBON,
+  GEL_SPACING,
+} from '../../../lib/constants/styles';
+import { T_MINION } from '../../../lib/constants/typography';
+
+const copyrightSymbolPrefix = '\u00A9';
+
+const StyledCopyright = styled.span`
+  ${T_MINION};
+  background-color: ${C_EBON};
+  color: ${C_WHITE};
+  padding: ${GEL_SPACING};
+  font-family: ${FF_NEWS_SANS_REG};
+  position: absolute;
+  bottom: 0;
+  right: 0;
+`;
+
+const Copyright = ({ text }) => (
```

Comment:
Can this be changed to `children` instead of `text` please. Therefore in the future if the copyright is a different lang to the rest of the document we can pass in the already handle HTML. EG: if this was an arabic company we may want to have the output 
```
© <span lang='ar'>صاحب حقوق الطبع والنشر</span>
```

Reply:
We would want to include the copyright character in that.

Do we just want to pass in the copyright character, along with the other children?

Would be easier to maintain.

Reply:
Then the Figure container is deciding what is rendered in the Copyright which doesn't feel right to me. Also would contradict how the the Caption appends it's own offscreen text within the component. Should try to be consistent whether a container or component is responsible to append data that only the component is concerned with

Reply:
@pjlee11 The copyright text on BBC sites is always translated into English, so we do not need to consider this. For a Persian article, we can have `<span lang="en-GB">STR</span>`. Note the screenshot of a Persian article with the English name of the copyright holder:
<img width="674" alt="screen shot 2018-09-06 at 17 02 21" src="https://user-images.githubusercontent.com/3028997/45170010-c6317000-b1f6-11e8-90ba-b80708aa2ec5.png">
http://www.bbc.com/persian/business-45420219

Reply:
Following @sareh's point my requested change is no longer relevant as the text will always be in English. I've opened an issue the issue https://github.com/bbc/simorgh/issues/623 in v1.1 to ensure we add the lang attribute for non-english articles.